src/pages/login/LoginForm.tsx

```diff
@@ -154,7 +155,7 @@ class LoginForm extends React.Component<LoginProps, State> {
 
   handleSubmit = (values: LoginFormInput, actions: FormikActions<LoginFormInput>) => {
     login(values.loginEmail, values.loginPassword).catch(error => {
-      actions.setErrors({ authenticationError: error.message });
+      actions.setErrors({ authenticationError: getFriendlyErrorMessage(error) });
```

Comment:
instead of authenticationError, you can name this `submitError` and it can be re-used as the attribute name for submission errors.

Reply:
👍